src/modules/communities/pages/CommunityPage/CommunityManagementPage/CommunityManagementPage.tsx

```diff
@@ -17,15 +25,102 @@ const CommunityManagementPage: React.FC = () => {
           setCommunity(res);
         })
         .catch(err => {
-          console.error(err);
+          toast.error(`${err} Erro ao carregar comunidade.`);
```

Comment:
O ideal seria ver como é retornado o erro. Se ele não for util para a UI eu removeria inclusive a impressão dele no toast. Deixando apenas: `toast.error('Erro ao carregar comunidade.');`

Reply:
show